pontoon/base/templates/translate.html

```diff
@@ -174,6 +174,7 @@
             {{ Filter.item('untranslated', 'Untranslated') }}
             {{ Filter.item('unchanged', 'Unchanged') }}
             {{ Filter.item('has-suggestions', 'Unreviewed Suggestions') }}
+            {{ Filter.item('rejected', 'Rejected Suggestions') }}
```

Comment:
Why add ``Suggestions`` here? I'm not sure I understand what you mean by ``suggestions``, but to me it's not a suggestion unless it's neither approved nor rejected. Does that make sense?